package.json

```diff
@@ -19,19 +19,16 @@
   "bugs": {
     "url": "https://github.com/IFS49F/poker/issues"
   },
-  "dependencies": {
-    "bootstrap": "^3.3.7",
-    "jquery": "^3.1.1",
-    "js-cookie": "^2.1.3",
-    "leancloud-storage": "^1.5.3",
-    "requirejs": "^2.3.2"
-  },
   "license": "MIT",
   "devDependencies": {
+    "bootstrap": "^3.3.7",
```

Comment:
Since travis will run `npm install` automatically, and we need to use webpack to generate prod files, I think it's OK.